### DISCUSSION TITLE
Are the Bootstrap 5 docs available for viewing?

### DISCUSSION THREAD

**Discussion by techboyg5** [twbs/bootstrap#31031] [General] [answered]
I can see the source files from the docs. However, do you know any place where you can view the docs for Bootstrap 5?

Also, when is Alpha 2 expected to come out?

**Comment by mdo** [comment, marked as answer]
http://twbs-bootstrap.netlify.app/ is the nightly build basically, which is all based on the pending v5 alpha.

First alpha is looking like this coming week, and a second alpha likely a few weeks to a month behind that. We're trying to get our release cadence more consistent. See #30602.

**Reply by techboyg5** [reply]
The first v5 alpha is already here by the way, look at the dist folder in the master branch.

**Reply by mdo** [reply]
I'm the one who created and maintains this, the first alpha has not yet arrived :). We've just changed the version name in `master` in preparation for the actual release.

**Reply by techboyg5** [reply]
> 
> 
> I'm the one who created and maintains this, the first alpha has not yet arrived :). We've just changed the version name in `master` in preparation for the actual release.

OK, thanks!
